docs/development/sync-protocol.md

```diff
@@ -356,12 +356,12 @@ downloading if the rules need to be downloaded in multiple batches.
 | Key | Required | Type | Meaning | Example Value |
 |---|---|---|---|---|
 | identifier | YES | string | The attribute of the binary the rule should match on e.g. the team ID of a binary or sha256 hash value | "ff2a7daa4c25cbd5b057e4471c6a22aba7d154dadfb5cce139c37cf795f41c9c" |
-| policy | YES | string | identifies the action to perform in response to the rule matching must be one of the examples. | "ALLOWLIST","ALLOWLIST_COMPILER", "BLOCKLIST", "REMOVE",  "SILENT_BLOCKLIST" |
-| rule_type | YES | string | identifies the type of rule must be one of he examples | "BINARY", "CERTIFICATE", "TEAMID" |
+| policy | YES | string | Identifies the action to perform in response to the rule matching. Must be one of the examples | "ALLOWLIST","ALLOWLIST_COMPILER", "BLOCKLIST", "REMOVE",  "SILENT_BLOCKLIST" |
+| rule_type | YES | string | Identifies the type of rule. Must be one of the examples | "BINARY", "CERTIFICATE", "SIGNINGID", "TEAMID" |
```

Comment:
Nit -  i appreciate making this more consistent without ending periods, however when multiple sentences are used it's awkward when the last one doesn't have a period. How about something like this:

```suggestion
| policy | YES | string | Identifies the action to perform in response to the rule matching (must be one of the examples) | "ALLOWLIST","ALLOWLIST_COMPILER", "BLOCKLIST", "REMOVE",  "SILENT_BLOCKLIST" |
| rule_type | YES | string | Identifies the type of rule (must be one of the examples) | "BINARY", "CERTIFICATE", "SIGNINGID", "TEAMID" |
```

Reply:
Much better! I mostly just wanted to add the extra rule type as it caught me out, then figured I’d tidy up the formatting a little but did not want to go mad!